Tool/src/integration/invalid_input.py

```diff
@@ -0,0 +1,4 @@
+import socket
```

Comment:
By the way you might consider bash or netcat instead of a whole python script

https://afterthoughtsoftware.com/posts/an-easy-way-to-send-udp-packets-in-linux
https://stackoverflow.com/questions/9696129/how-to-send-only-one-udp-packet-with-netcat

Reply:
haven't figured out string format if using `> /dev/udp/127.0.0.1/2000`, but `nc` works.